controllers/gitrepo/gitrepo.go

```diff
@@ -0,0 +1,11 @@
+package gitrepo
+
+import (
+	"github.com/go-git/go-git/v5"
+)
+
+// Clone is equal to `git clone`
+func Clone(url, path string) error {
+	_, err := git.PlainClone(path, false, &git.CloneOptions{URL: url})
+	return err
```

Comment:
How this file and func are used?

Reply:
It will be used in this function https://github.com/oam-dev/terraform-controller/pull/110/files#diff-7c1f23f62bf3f0bdba1d953b9651e57b9c656e6e824f75d3a5004fdb25e5a735R153. Currently it's not used.

Reply:
alright, got it.

Reply:
LGTM